reverted image style

illudedPerception opened the pull request:
* Reverted water occupacity back to 100% from 50%
* Added back height numbers to maps what had them
* Added heights to mesa plains, height accuracy is ~2-3m cause of rounding and data manipulation in the raster
* Standardized height number fonts (few fonts were impact instead of open sans)
* Didin't change the map borders
* Added back non rotated images

transform data for non rotated images
PineValley.png
92.95,0,0,-92.95,-199953.52,199953.52
LakeValley.png
91.88,0,0,-91.88,-197454.05,197454.05
AuroraFalls.png
139.91,0,0,-139.91,-300900.04,300460.04

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Summary by CodeRabbit

- **New Features**
	- Updated background images and transformation matrices for the 'MesaPlains', 'AuroraFalls', and 'LakeValley' game levels.
	- Enhanced border dimensions to align with the new background settings.

- **Bug Fixes**
	- Improved visual consistency of the map based on the selected game level.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

illudedPerception committed:
reverted image style

* Reverted water occupacity back to 100% from 50%
* Added back height numbers to maps what had them and can have them
* Didint change the map borders

illudedPerception committed:
small fixes

*standardized fonts
*added heights to mesa plains, height accuracy is ~2-3m cause of rounding and data manipulation

illudedPerception committed:
Update RailroadMap.ts

scottanderson commented:
We will still need the old inverted map for Lake Valley. Please add the new files without removing the old ones.

illudedPerception committed:
Update RailroadMap.ts

illudedPerception committed:
Delete railroad.studio/PineValleyTopo.png

illudedPerception committed:
Delete railroad.studio/AuroraFallsTopo.png

illudedPerception committed:
Delete railroad.studio/LakeValleyTopo.png

illudedPerception committed:
Delete railroad.studio/MesaPlainsTopo.png

illudedPerception committed:
Add files via upload

illudedPerception committed:
old images

transform data
Pine Valley
92.95,0,0,-92.95,-199953.52,199953.52
Lake Valley
91.88,0,0,-91.88,-197454.05,197454.05
Aurora Falls
139.91,0,0,-139.91,-300900.04,300460.04
